.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Insufficient secret file exclusion patterns</b></div>
<div id="fix">

Consider adding sensitive file patterns like '*.pem', '*.key', '*.env', 'credentials.*' to prevent accidental commit of secrets and credentials. The '.idea' entry alone may not sufficiently protect against exposing sensitive configuration data.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
 .idea
 *.pem
 *.key
 *.env
 credentials.*
 *secret*
 .env*
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/36#issuecomment-2557127616>#affd6a</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged